pyconbalkan/timetable/models.py

```diff
@@ -30,6 +36,11 @@ class Presentation(AbstractConference, ActiveModel):
     def slugify(self):
         return slugify(self.title)
 
+    @property
```

Comment:
no, this method should be on the speaker model, not here !

Reply:
so what blocks you from saying session.speaker.slugify, why do you need to call session.slugify_speaker?